tests/queries/0_stateless/02723_zookeeper_name.sql

```diff
@@ -0,0 +1,23 @@
+-- Tags: zookeeper, replica
+
+SELECT 'Create Tables';
+CREATE TABLE t1(k UInt32, v UInt32) ENGINE ReplicatedMergeTree('/clickhouse/tables/{database}/test_02723/zookeeper_name/t1', '1') ORDER BY k;
+
+CREATE TABLE t2(k UInt32, v UInt32) ENGINE ReplicatedMergeTree('/clickhouse/tables/{database}/test_02723/zookeeper_name/t2', '1') ORDER BY k;
+
+SELECT 'Insert Data';
+
+INSERT INTO t1 SELECT * FROM generateRandom('k UInt32, v UInt32') LIMIT 1;
+INSERT INTO t2 SELECT * FROM generateRandom('k UInt32, v UInt32') LIMIT 1;
+
+SELECT
```

Comment:
That requst is flaky
Fail in master, for example.
https://s3.amazonaws.com/clickhouse-test-reports/35827/248e8819ff2c5fb9b78f32f8f0199f77c9a841b0/stateless_tests__debug__[5_5].html

Order by needed.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/49592